.github/workflows/_release.yml

```diff
@@ -35,6 +35,24 @@ jobs:
         id: install
       - run: yarn test --coverage
       - uses: codecov/codecov-action@b9fd7d16f6d7d1b5d2bec1a2887e65ceed900238 # v4
+      - name: Update eslint-ci-rules if needed
+        if: contains(github.event.head_commit.message, 'update dependency eslint-config-txo')
+        run: |
+          if ! yarn lint:ci; then
+            echo "Linting failed. Updating eslint-ci-rules.json..."
+            yarn txo-eslint update-ci-rules
+            changed_files=$(git diff --name-only)
+            if [[ "$changed_files" == "eslint-ci-rules.json" ]]; then
+              git config --global user.email "txo-github-bot[bot]@users.noreply.github.com"
+              git config --global user.name "TXO GitHub Bot[bot]"
+              git remote set-url origin https://x-access-token:${{ steps.get-token.outputs.token }}@github.com/${{ github.repository }}
+              git add eslint-ci-rules.json
+              git commit -m "chore: update eslint-ci-rules.json [skip ci]"
+              git push
+            fi
+          else
+            echo "Linting passed. No update needed."
+          fi
```

Comment:
Are we sure that semantic release will succeed in publishing if there will be new commit present ?